runner/serverhandler.go

```diff
@@ -39,12 +39,12 @@ func (r *Runner) RegisterWithGocast(retries int) {
 	}
 }
 
-// dialIn connects to manager instance and returns a client
-func (r *Runner) dialIn() (protobuf.RunnerManagerServiceClient, error) {
+// DialIn connects to manager instance and returns a client
+func DialIn() (protobuf.RunnerManagerServiceClient, error) {
```

Comment:
Why is this change neccessary? I don't like that this method becomes part of the API surface of the `runner` package. Is it to be able to use the method in onPublish [1]?
If so, I'd suggest making onPublish a member function of Runner to avoid having to export this.

[1]: https://github.com/TUM-Dev/gocast/blob/a6a86eb11465b5383ecd4122c054781397452c7e/runner/rest.go#L53